app/src/main/contrib/components/Responsibilities/Overview/List/CurrentEstimateSetSummary.tsx

```diff
@@ -58,10 +58,10 @@ export class CurrentEstimateSetSummary extends React.Component<CurrentEstimateSe
         return <div className={`mt-3 alert alert-${alertType} text-left`}>
             {estimateText}
             {url && [
-                <FileDownloadButton href={url} className={"float-right"}>
+                <FileDownloadButton href={url} className={"float-right"} key={`download-${estimateSet.id}`}>
```

Comment:
I noticed React was complaining that you need to use a key when rendering an array of elements as here